synfig-studio/src/gui/dialogs/dialog_guide.cpp

```diff
@@ -160,4 +195,6 @@ Dialog_Guide::init_widget_values()
 		angle_widget->set_value(synfig::Angle::deg(curr_guide->angle).get());
 	else
 		angle_widget->set_value(curr_guide->angle.get());
+	x_widget->set_value((curr_guide->point[0])*60);
+	y_widget->set_value(curr_guide->point[1]*60);
```

Comment:
The magic number `60` is true for the default canvas settings. To get the actual value, you have to look to `RendDesc` value of current `Canvas`.

Reply:
By default do you mean the aspect ratio, physical dimensions? im confused as to what im supposed to use from renddesc i see resolutions, height and width but the workarea ruler doesnt seem to be affected by these properties what property exactly should i be working with here?

Reply:
@rodolforg 
```RendDesc rend_desc = rend_desc();``` 
```float factor = std::fabs((rend_desc.get_w()/2)/rend_desc.get_tl()[0]);```